xml/templates/message_templates/participant_cancelled_text.tpl

```diff
@@ -7,10 +7,10 @@
 {ts}Event Information and Location{/ts}
 
 ===========================================================
-{$event.event_title}
-{$event.event_start_date|crmDate}{if $event.event_end_date}-{if $event.event_end_date|date_format:"%Y%m%d" == $event.event_start_date|date_format:"%Y%m%d"}{$event.event_end_date|crmDate:0:1}{else}{$event.event_end_date|crmDate}{/if}{/if}
+{event.title}
+{$event.event_start_date|crmDate}{if $event.event_end_date}-{if $event.event_end_date|crmDate:"shortdate" == $event.event_start_date|crmDate:"shortdate"}{$event.event_end_date|crmDate:"Time"}{else}{$event.event_end_date|crmDate}{/if}{/if}
```

Comment:
In the html version the `$event.event_start_date` was changed to `{event.start_date}` but not here.

Reply:
opps - I thought I pulled out those - the last changes Tim made in the rc broke event tokens again I think - I was gonna dig into that after this was merged but I probably only removed from the html 


FYI @magnolia61 - I think we had some regressions around even tokens in the rc - @totten ripped out the bit where the participant token class ensured the event id was set. It passed tests (which should have ensured that didn't break anything) but is failing here..... not sure if you want to do any re-testing. I think it's regressed from earlier in 5.43 rather than from 5.42

Reply:
Just there's a mismatch still between the text and html changes.

Reply:
the only mismatch I am still seeing here is that in the text we are now using crmDate instead of date_format unlike in the html version in L38 @eileenmcnaughton